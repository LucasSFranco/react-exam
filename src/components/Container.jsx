import styled from 'styled-components'

const Container = styled.div`
  margin: 0 1.5rem;

  @media (min-width: 768px) {
    margin: 0 3rem;
  }

  @media (min-width: 1280px) {
    max-width: calc(1280px - 3rem);

    margin: 0 auto;
  }
`

export default Container
