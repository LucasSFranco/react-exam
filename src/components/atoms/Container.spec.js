import React from 'react'
import { render } from '@testing-library/react'

import Container from './Container'

describe('Container', () => {

  it('properly renders', async () => {

    const { container } = render(
      <Container>
        Children
      </Container>
    )

    expect(container.firstChild).toMatchSnapshot()
  })

})
