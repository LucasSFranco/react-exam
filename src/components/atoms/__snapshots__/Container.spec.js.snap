// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container properly renders 1`] = `
<div
  class="sc-bdnxRM jrPDBJ container"
>
  Children
</div>
`;
